book/prover-network/setup.md

```diff
@@ -6,30 +6,43 @@ Succinct [has been building](https://blog.succinct.xyz/succinct-network/) the Su
 
 ## Get access
 
-Currently the network is permissioned, so you need to gain access through Succinct. After you have completed the key setup below, you can submit your address in this [form](https://docs.google.com/forms/d/e/1FAIpQLSd-X9uH7G0bvXH_kjptnQtNil8L4dumrVPpFE4t8Ci1XT1GaQ/viewform) and we'll contact you shortly.
+**[FILL OUT THIS FORM](https://forms.gle/rTUvhstS8PFfv9B3A)** to gain access to the Succinct
```

Comment:
IMO this was better as it was before

Reply:
@ratankaliani said that it was too hard to find the form